.travis.yml

```diff
@@ -11,3 +11,7 @@ rvm:
   - 1.9.3
   - jruby-19mode
   - rbx-2.2.10
+
+matrix:
+  allow_failures:
+    - rvm: rbx-2.2.10
```

Comment:
Change both references or just this `allow_failures` reference? Sorry, I'm not very familiar with travis configuration.